src/Processor/ForeachProcessor.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Elastica\Processor;
+
+/**
+ * Elastica Attachment Processor.
+ *
+ * @author Federico Panini <fpanini@gmail.com>
+ * @author Thibaut Simon-Fine <tsimonfine@gmail.com>
+ *
+ * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/foreach-processor.html
+ */
+class ForeachProcessor extends AbstractProcessor
+{
+    public const DEFAULT_IGNORE_MISSING_VALUE = false;
+
+    /**
+     * Foreach constructor.
+     */
```

Comment:
Doesn't bring much value, could probably be removed

Reply:
Resolved in https://github.com/ruflin/Elastica/pull/1813/commits/e7659be9d92078ad97a48bd994f580eaa01abb1a